tests/playwright/outside/api.spec.js

```diff
@@ -6,6 +6,34 @@ const { describe } = test;
 
 const API_ENDPOINT = "http://localhost:9080/jsonapi";
 
+// helper function to upload files to the JSON API
+const uploadFile = async (request, where, filename) => {
+  const binaryData = await fs.readFileSync(
```

Comment:
```suggestion
  const binaryData = fs.readFileSync(
```

It's a synchronous function, nothing to await. :)

Reply:
d'oh!